cypress/integration/logistics/create_new_tour_and_transportation_order_spec.js

```diff
@@ -17,14 +18,27 @@ describe('create new tour and transportation order', function() {
         .setName(tourName)
         .apply();
     });
+    cy.getStringFieldValue('AD_Org_ID').then(organisation => {
+      shipperBPartner = organisation;
+    });
+  });
+
+  it('Create a shipper', function() {
+    cy.fixture('settings/shipper.json').then(shipperJson => {
```

Comment:
q: what is a shipper, and why is it in the `settings` folder?

Reply:
I would guess that a shipper is  the one who ships orders (company that transports or receives goods by sea, land, or air.), and it's under settings because the window in the app it's also under settings and I thought that's the place where it should be
![image](https://user-images.githubusercontent.com/50820854/62532571-59cb5080-b84d-11e9-908c-79f1bd8529e5.png)

Reply:
You mean like a courier (DHS or FanCourier)?

Reply:
Yes.